tests/dialects/test_duckdb.py

```diff
@@ -711,6 +723,10 @@ def test_array_index(self):
                     "WARNING:sqlglot:Applying array index offset (1)",
                     "WARNING:sqlglot:Applying array index offset (-1)",
                     "WARNING:sqlglot:Applying array index offset (1)",
+                    "WARNING:sqlglot:Applying array index offset (-1)",
+                    "WARNING:sqlglot:Applying array index offset (1)",
+                    "WARNING:sqlglot:Applying array index offset (-1)",
+                    "WARNING:sqlglot:Applying array index offset (1)",
```

Comment:
I think this assertion is confusing, the sequence by itself doesn't explain much besides the fact that there are a bunch of warnings logged. I'd rather either 1) get rid of it altogether (so won't need the `as cm` part above), or 2) reduce the scope of the assertion to only a single test so that we observe the dec/increment of the index once.

Reply:
By changing the array indexes to be an identifier instead of an integer literal we can get rid of these 4 warnings and maintain the tests, would that work?

Reply:
I'd just do (2) instead of augmenting this list every time we add an we test tbh